### PR TITLE
updating code for HA redis

### PR DESCRIPTION
**Pull request by LindsayYoung**
I already created the high availability redis32 instances in dev, stage and prod.

After successful deploy, we will what to delete the old redis instance in each environment.

### CODE REVIEW

**Comment by vrajmohan** [comment]
Would this be a good time to take another step in https://github.com/18F/openFEC/issues/2247?

**Comment by LindsayYoung** [comment]
sure, I can do that

**Comment by LindsayYoung** [comment]
remade the redis instances with the new name and updated code

Thanks @vrajmohan

File: manifests/manifest_dev.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ inherit: manifest_base.yml
 host: fec-dev-api
 services:
   - fec-creds-dev
-  - fec-redis-dev
+  - fec-redis
```
Comment (vrajmohan) [inline manifests/manifest_dev.yml]: Sorry to nitpick, they can all go in `manifest_base` now, as they are all the same.

Comment (LindsayYoung) [reply manifests/manifest_dev.yml]: @vrajmohan updated :)

File: manifests/manifest_stage.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ inherit: manifest_base.yml
 host: fec-stage-api
 services:
   - fec-creds-stage
-  - fec-redis-stage
+  - fec-redis
```
Comment (vrajmohan) [inline manifests/manifest_stage.yml]: Looks like you missed a spot!

**Pull request merged**